changelog.txt

```diff
@@ -29,14 +29,16 @@ that repo.
 - `makeown`: adds the selected unit as a member of your fortress
 
 ## Fixes
--@ `gui/unit-syndromes`: allow the window widgets to be interacted with
--@ `fix/protect-nicks`: now works by setting the historical figure nickname
--@ `gui/liquids`: fixed issues with unit pathing after adding/removing liquids
+- `gui/unit-syndromes`: allow the window widgets to be interacted with
+- `fix/protect-nicks`: now works by setting the historical figure nickname
+- `gui/liquids`: fixed issues with unit pathing after adding/removing liquids
+- `gui/create-item`: adding corpse spawning broke spawning generic corpsepieces, but has now been fixed.
```

Comment:
is this a merge error? these should all be prefixed with `-@`

Reply:
oops i did that manually and didn't know the format